.github/ISSUE_TEMPLATE/1.general_issue_report.yaml

```diff
@@ -0,0 +1,77 @@
+name: Issue Report
+description: File an issue report
+title: "[Issue]: "
+labels: ["bug", "status:triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue report! Please ensure you've read our [documentation](https://docs.hydra.so/). If you're unsure as to whether or not you're experiencing an issue, please reach out on our [Discord](https://discord.com/invite/SQrwnAxtDw) to gain a better understanding of what you're facing.
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: First check
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title to this issue.
+          required: true
+        - label: I used the GitHub search to find a similar issue and didn't find it.
+          required: true
+        - label: I searched the Hydra documentation for this issue.
+          required: true
+        - label: I checked that this issue is related to Hydra and not one of its dependencies.
+          required: true
+
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
```

Comment:
Not sure if we need these, we don't want to add too much weight to the process yet.

Reply:
I feel like this might be too "heavy" at the moment.  if we do keep this, then we should lose the checkboxes and contact details.  if it becomes a problem in the future, we can revisit, but we're not getting that much traffic on issues.

I think it might be good to keep the documentation and discord links, wdyt @wuputah ?